packages/react/src/_popover/Popover.tsx

```diff
@@ -44,45 +46,65 @@ const DEFAULT_OFFSET = 8;
 type UsePopoverOptions = {
 	/** The placement of the popover element in relation to the reference element. */
 	placement?: Placement;
+	/** The minimum acceptable height of the popover element before the `flip` middleware takes over.  */
+	minHeight?: number;
 	/** The maximum height of the popover element. */
 	maxHeight?: number;
 	/** If true, the popover element will match the width of the reference element. */
 	matchReferenceWidth?: boolean;
 	/** Used to control the vertical distance between the reference element and popover element. Value is in pixels. */
 	offset?: number;
+	/** If true, the popover element is open. Required when using the `hiddenWithCSS` option. */
+	isOpen?: boolean;
+	/** If true, the popover element is using `display: none` or `visibility: hidden` instead of conditional rendering. */
+	hiddenWithCSS?: boolean;
 };
 
 export function usePopover<RT extends ReferenceType = ReferenceType>(
 	options?: UsePopoverOptions
 ) {
 	const {
+		isOpen,
+		hiddenWithCSS = false,
 		placement = 'bottom-start',
 		matchReferenceWidth = false,
-		maxHeight,
+		minHeight: minHeightOption,
+		maxHeight: maxHeightOption,
 		offset: offsetOption = DEFAULT_OFFSET,
 	} = options || {};
 
-	const { refs, floatingStyles } = useFloating<RT>({
+	const floating = useFloating<RT>({
 		placement,
 		middleware: [
 			// Adds distance between the reference and floating element
 			// https://floating-ui.com/docs/offset
 			offset(offsetOption),
 			// Changes the placement of the floating element in order to keep it in view
 			// https://floating-ui.com/docs/flip
-			flip(),
+			flip({ padding: DEFAULT_OFFSET }),
```

Comment:
This has updated fixed after another read through the Floating UI documentation: https://floating-ui.com/docs/size#using-with-flip

> If you’re using the padding option in either middleware, ensure they share the same value.